src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -2,6 +2,41 @@
 
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder salaryInfo = new StringBuilder("Report for period " + dateFrom + " - "
+                + dateTo);
+        int[] fromDate = dateToIntArray(dateFrom);
+        int[] toDate = dateToIntArray(dateTo);
+        for (String employeeName : names) {
+            int totalSalary = 0;
+            for (String employeeData : data) {
+                String[] splittedData = employeeData.split(" ");
+                int[] workDate = dateToIntArray(splittedData[0]);
+                if (employeeName.equals(splittedData[1]) && dateFitsPeriod(workDate, fromDate,
+                        toDate)) {
+                    totalSalary += Integer.parseInt(splittedData[2])
+                            * Integer.parseInt(splittedData[3]);
```

Comment:
https://github.com/mate-academy/jv-salary-info/blob/master/checklist.md#any-magic-numbers-should-be-constants

Reply:
Those aren't a "magic" numbers, those are array indices

Reply:
they are too obvious to make constants